README.md

```diff
@@ -292,19 +288,19 @@ If you want to automatically ignore certain directories when terraform-ls indexe
 You can configure the path to the Terraform binary used by terraform-ls to perform operations inside the editor by configuring this setting:
 
 ```json
-"terraform-ls.terraformExecPath": "C:/some/folder/path"
+"terraform.languageServer.terraform.executable.path": "C:/some/folder/path"
 ```
 
 You can override the Terraform execution timeout by configuring this setting:
 
 ```json
-"terraform-ls.terraformExecTimeout": "30"
+"terraform.languageServer.terraform.executable.timeout": "30"
 ```
 
 You can set the path Terraform logs (`TF_LOG_PATH`) by configuring this setting:
 
 ```json
-"terraform-ls.terraformLogFilePath": "C:/some/folder/path/log-{{varname}}.log"
+"terraform.languageServer.terraform.executable.logFilePath": "C:/some/folder/path/log-{{varname}}.log"
```

Comment:
We went back and forth with lots of variations and left it with this naming to be explicit that this was how to configure how the CLI behaves and where it was physically located. 

Removing `executable` results in `terraform.languageServer.terraform.path` and `Language Server > Terraform > Path`. My concern is that `terraform.languageServer.terraform.path` and `terraform.languageServer.terraform.logFilePath` do not have much visually to disambiguate each other, but I may be over thinking it.

However, if we can conceivably add more options here, then it would make sense to remove `executable` as I think this reads nicely:

```
terraform.languageServer.terraform.path
terraform.languageServer.terraform.timeout
terraform.languageServer.terraform.plan.parallelism
terraform.languageServer.terraform.apply.autoApprove
```

This allows segments for each subcommand and room for growth without squishing everything together like `terraform.languageServer.terraform.planParallelism`.

Reply:
Removed `executable`